types/mux.d.ts

```diff
@@ -233,7 +233,7 @@ export declare interface FilterGetResponse {
 }
 
 export declare class Filters extends Base {
-  list: Promise<FiltersListResponse>;
```

Comment:
ahh, looks like this should be `list(): Promise<FiltersListResponse>;`

https://muxinc.github.io/mux-node-sdk/class/src/data/resources/filters.js~Filters.html

`Data.Filters.list()` is a function that returns a `Promise` that resolves with `FiltersListResponse`

Does that look right?

Reply:
Oh yeah, my bad, didn't see it was a class - that looks right